source/NVDAObjects/window/winword.py

```diff
@@ -1323,6 +1323,15 @@ def script_reportCurrentComment(self,gesture):
 	# Translators: a description for a script
 	script_reportCurrentComment.__doc__=_("Reports the text of the comment where the System caret is located.")
 
+	def script_lineSpacing(self,gesture):

```

Comment:
Probably best to add a verb to the name 
of the script, such as changeLineSpacing, as opposed to just lineSpacing.